pkg/epinio/detail/applications.vue

```diff
@@ -202,49 +237,94 @@ export default Vue.extend<Data, any, any, any>({
       h4:last-of-type {
         word-break: break-all;
       }
-
       &:last-of-type {
         h4:last-of-type {
           margin-bottom: 0;
         }
       }
+      thead {
+        tr {
+          th {
+            text-align: left;
+            color: #c4c4c4;
```

Comment:
We should avoid using colours directly in components. These are generally brittle and can also break light/dark mode. 
Colours can be applied via scss variable from `/shell/assets/styles/themes/_light.scss` (the same folder contains overrides for suse and dark themes)